# 4host


Microarray experiment, comparing P. capsici gene expression on four different host species. 
Infected leaf material was sampled at 16 hours post inoculation and analised using an agilent array

#####Create probeset
Probeset was generated by using all known P. capsici genes (Phyca11) and additionally, all in house predicted RxLR and CRN effectors.
Overlaps (e.g. a gene annotated in Phyca11 and our predictions) were removed and replaced by our effectors using `a script`

As input we used gff files from the genome assembly and our own predicted effectors:  `Phyca11.gff` (as published by Lamour et al 2013), `CRN.gff` (as published by Stam et al 2013), and  `RxLR.gff` (In house prediction)

The resulting fasta file `Trimmed_Phyca11_effector.fasta` was used in Agilent's earray software https://earray.chem.agilent.com/earray/ to generate the probeset. All earray files can be found in `Arrayexpress` [here](http://www.emptylink.html)


#####Microarray analysis
The microarray data was analysed using `limma` from the `bioconductor` package. [here](http://www.bioconductor.org/packages/release/bioc/html/limma.html)

All steps taken during this analysis are recorded in R Markdown files that can be found on this github.
The resulting html files can also be found here [here](http://www.compbio.dundee.ac.uk/user/rstam/internal/R_markdown/4Host/)


